db/tables.py

```diff
@@ -52,6 +55,42 @@ def create_mathesar_table(name, schema, columns_, engine, metadata=None):
     return table
 
 
+class DropTableCascade(DropTable):
+    def __init__(self, table, cascade=False, if_exists=False, **kwargs):
+        print(if_exists)
+        super().__init__(table, if_exists=if_exists, **kwargs)
+        self.cascade = cascade
+
+
+@compiler.compiles(DropTableCascade, "postgresql")
+def compile_drop_table(element, compiler, **_):
+    expression = compiler.visit_drop_table(element)
+    if element.cascade:
+        return expression + " CASCADE"
+    else:
+        return expression
+
+
+def delete_table(name, schema, engine, cascade=False, if_exists=False):
```

Comment:
If `cascade` is `False`, can you return a list of related items just like with schemas?